Added hint for non-root linking of yotta packets

meriac opened the pull request.

autopulated commented:
Thanks! A few things:

 * I'm preferring "module" over packet or package, as those words have too many other connotations.
 * you both install, and remove python-pip, is that right?
 * get_pip.py presumably needs to be obtained from somewhere?

meriac commented:
fixed:
- changed wording from package to module
- yes, first the outdated pip package needs to be removed, then the updated package is installed manually
- fixed  get_pip.py download
